### DISCUSSION TITLE
./setup_xml.py  error on cactus for HR4

### DISCUSSION THREAD

**Discussion by jiandewang** [NOAA-EMC/global-workflow#3086] [General]
I have two identical HR4 setup on two wcoss2 which I used to run HR4 in the past without any issue, but since the machine maintance ~10 days ago I am unable to run setup_xml.pl on cactus (but it still works fine on dogwood at this moment).

 ./setup_expt.py gfs forecast-only --app S2SW --pslot HR4-20191215 --icsdir /lfs/h2/emc/global/noscrub/emc.global/data/ICSDIR/C1152mx025/20240829 --configdir /lfs/h2/emc/couple/noscrub/Jiande.Wang/UFS-HR4/global-workflow/parm/config/gfs --idate 2019121500 --edate 2019121500 --resdetatmos 1152 --resdetocean 0.25 --gfs_cyc 1 --comroot /lfs/h2/emc/ptmp/jiande.wang/work-HR4/HR4-20191215/COMROOT/2019121500 --expdir /lfs/h2/emc/ptmp/jiande.wang/work-HR4/HR4-20191215/EXPDIR/2019121500
Traceback (most recent call last):
  File "./setup_expt.py", line 13, in <module>
    from hosts import Host
  File "/lfs/h2/emc/couple/noscrub/Jiande.Wang/UFS-HR4/global-workflow/workflow/hosts.py", line 7, in <module>
    from wxflow import YAMLFile
  File "/lfs/h2/emc/couple/noscrub/Jiande.Wang/UFS-HR4/global-workflow/workflow/wxflow/__init__.py", line 13, in <module>
    from .jinja import Jinja
  File "/lfs/h2/emc/couple/noscrub/Jiande.Wang/UFS-HR4/global-workflow/workflow/wxflow/jinja.py", line 16, in <module>
    @jinja2.pass_eval_context

my HR4 g-w on cactue is at /lfs/h2/emc/couple/noscrub/Jiande.Wang/UFS-HR4/global-workflow

two other users also found the same issue (works fine on dogwood but not on cactus)

**Comment by WalterKolczynski-NOAA** [comment]
Yeah, we're seeing this in PR testing as well. Something changed during the upgrade that broke things.

**Reply by aerorahul** [reply]
https://github.com/NOAA-EMC/global-workflow/issues/3087 was created to report this issue, and https://github.com/NOAA-EMC/global-workflow/pull/3053 resolved it.
